CONTRIBUTING.md

```diff
@@ -8,16 +8,22 @@ Conduct].
 
 1. Fork the repo.
 
-2. Install [dependencies](#installing-dependencies).
+2. Install dependencies by running `bundle install`.
+If you're using Linux, you may need to [install certain
+packages](#installing-dependencies) to set up a working development
+environment.
 
-3. Run the tests. We only take pull requests with passing tests, and it's great
-to know that you have a clean slate: `bundle && bundle exec rake`
+3. All tests should be passing, but it's a good idea to run them anyway
+before starting any work: `bundle exec rake`
 
 4. If you're adding functionality or fixing a bug, add a failing test for the
 issue first.
 
 5. Make the test pass.
```

Comment:
(from Elliot) Maybe this should include instructions for running a single spec file since that's a little non-standard?